provider/lxd/credentials.go

```diff
@@ -111,7 +111,16 @@ func (environProviderCredentials) CredentialSchemas() map[cloud.AuthType]cloud.C
 }
 
 // RegisterCredentials is part of the environs.ProviderCredentialsRegister interface.
-func (p environProviderCredentials) RegisterCredentials() (map[string]*cloud.CloudCredential, error) {
+func (p environProviderCredentials) RegisterCredentials(credentialName string) (map[string]*cloud.CloudCredential, error) {
```

Comment:
Isn't this "cloudName" not "credentialName" ?
"Register the Credentials for the cloud named X"

But also, why are we restricting it to just the cloud Name, couldn't we put in the full Cloud definition, and then in the future we might do something special based on, say, the Auth URL or something like that?

Reply:
@jameinel that's not a bad idea, I will change that!